.github/workflows/omni.yml

```diff
@@ -2,6 +2,36 @@ name: Omni CI
 
 on:
   workflow_dispatch:
+    inputs:
+      gradle_args:
+        description: Gradle arguments
+        required: false
+        default: "jandex test jacocoReport"
+      # Filters to limit generated matrix. Empty or "*" values are ignored.
+      matrix_jdk_distribution:
+        description: JDK Distribution (adopt, zulu, ...)
+        required: false
+        default: "*"
```

Comment:
what does * mean here, all of the distros ?

Reply:
Yes. Empty string or "*" (star) match everything for any filter. I added the star value as it looks a bit cleaner in the UI.